pkg/ccl/importccl/import_stmt.go

```diff
@@ -702,10 +701,10 @@ func parseAvroOptions(
 }
 
 type importResumer struct {
-	job            *jobs.Job
-	settings       *cluster.Settings
-	res            roachpb.BulkOpSummary
-	statsRefresher *stats.Refresher
+	job      *jobs.Job
+	settings *cluster.Settings
+	res      roachpb.BulkOpSummary
+	execCfg  *sql.ExecutorConfig
```

Comment:
I'd have a slight preference for passing these to the helper directly instead of mutating `r` to stick them on it here and then just assuming they're on `r` later, but it is slight.